docs/contributors/code/release.md

```diff
@@ -62,17 +73,23 @@ Documenting the release is a group effort between the release manager, Gutenberg
 
 #### 1. Curating the changelog
 
-The release notes draft is auto-generated by a script that looks for pull requests for the current milestone, and groups them by pull request label.
+The release notes draft is auto-generated by a script that looks for pull requests for the current milestone, and:
+
+1. Groups them into sections (by pull request label).
+2. Sub-groups them into "features" within each section (again by pull request label)
+
 This is intended to be a starting point for release notes, and manually reviewing and curating the changelog entries is still required. The release candidate changelog is reused in the stable release and greatly helps select the highlights; because depending on the release it can be a very time-consuming process, **it is recommended to start this process as soon as the milestone is closed** and the release candidate is published.
 
 Guidelines for proof-reading include:
 
+-   Move _all_ entries under the `Various` section to a more appropriate section.
+-   Move _all_ features listed under the `Uncategorized` bullet points to a more suitable feature grouping.
 -   Fix spelling errors or clarify wording. Phrasing should be easy to understand where the intended audience is those who use the plugin or are keeping up with ongoing development.
 -   Create new groupings as applicable, and move pull requests between.
 -   When multiple pull requests relate to the same task (such as a follow-up pull request), try to combine them to a single entry.
 -   If subtasks of a related set of pull requests are substantial, consider organizing as entries in a nested list.
 -   If one or more pull requests revert one or more pull requests in the same release netting a zero-sum of code changes, remove them.
--   Remove mobile app pull request entries.
+-   Remove all mobile app pull request entries.
```

Comment:
```suggestion
-   Remove all mobile app pull request entries.
```

There is one exception, if the mobile app related PR updates also funcionality for the web. This is how we missed React upgrade from v16 to v17 in the plugin changelog 😅

Reply:
I've updated the docs to reflect this 👍